helm/values/dev-main.yaml

```diff
@@ -7,7 +7,7 @@ config:
   ## this only take the image form URL
   chain_logo_url: "https://storage.googleapis.com/voice_logo/logo.ico"
   es_replicas: 1
-  abi_scan: false
+  abi_scan: true
```

Comment:
would these changes need to be in all the helm chart files for each env?

Reply:
this is to repeat the hyperion as discussed with both Roman and Bohdan